tutorial01_load_vulkan/app/build.gradle

```diff
@@ -2,7 +2,7 @@ apply plugin: 'com.android.application'
 
 android {
     compileSdkVersion 24
-    buildToolsVersion '25.0.0'
+    buildToolsVersion '26.0.2'
```

Comment:
Starting with AGP 3.0.0, this property is optional, and should be removed from our samples.

Reply:
done